### ISSUE TITLE
Further question about IIIF image

### ISSUE THREAD

**Issue by dancorpse** [lovasoa/dezoomify#56]
For this site: http://www.bavarikon.de/object/bav:BSG-OBJ-0000000000000117, I think it's IIIF image so I followed this article: https://github.com/lovasoa/dezoomify/wiki/How-to-download-full-resolution-images-from-an-IIIF-compatible-server, and the full image url should be like: http://www.bavarikon.de/iiif/BSG-OBJ-0000000000000117/full/full/0/native.jpg, but the images do not showed.
Am I anything wrong? 
Thanks.

**Comment by lovasoa** [comment]
Yes, they forbid access to full image. This means we may have to write a dezoomifier even for IIIF.

**Comment by lovasoa** [comment]
Last commit added support for IIIF: 23624e2189a8e582ac9ca1e1d2a4d4dc1a1166eb

But this site seems to be checking referrer when delivering tiles, and thus still doesn't work.

**Comment by navydinosaur** [comment]
I hope I'm not posting this in the wrong place, but I think my issue is related. Is this site also checking referrer?
http://maps.nls.uk/imgsrv/iipsrv.fcgi?iiif=/10324/103247211.jp2/full/full/0/native.jpg

If I select the IIIF option, I get the "Error: No IIIF URL found" message. If I try the automatic option, I get the "Error: Unable to find a proper dezoomer for the given URL" message. I've also tried the script unsuccessfully, and can give details wherever it's appropriate.

Thanks for any help.

**Comment by lovasoa** [comment]
## How to use dezoomify with IIIF

When your image displays, look at the network requests, and find the small json meta-information file that should be loaded first. It is that file that you should give to dezoomify.

If you can't find it from your network requests, you can reconstruct its URL. Just add `/info.json` after the file name in the URL.

In your case:

`http://maps.nls.uk/imgsrv/iipsrv.fcgi?iiif=/10324/103247211.jp2/info.json`

**Comment by navydinosaur** [comment]
Thank you very much for the quick response, and sorry for the confusion on my end! I was following the instructions on this page: https://github.com/lovasoa/dezoomify/wiki/How-to-download-full-resolution-images-from-an-IIIF-compatible-server and must have misunderstood the URL scheme.

The image successfully loaded with the addition of "/info.json", but in Safari there was no option to save it, in Firefox I only saw a blank page (which I tried to fix with the 'max-alloc-size' change -- unsuccessfully), and in Chrome, nothing happened when I selected "Save image as". Is there any way to save a successfully dezoomed and loaded image if none of my browsers are working?

Thanks again.

**Comment by lovasoa** [comment]
Bot-assisted investigation note:

I checked the URLs from this issue against the current site behavior.

The guessed IIIF URL in the original report is no longer valid:

- `http://www.bavarikon.de/iiif/BSG-OBJ-0000000000000117/full/full/0/native.jpg` now returns 404.

The current working IIIF Image API metadata endpoint for that object appears to be:

- `https://api.digitale-sammlungen.de/iiif/image/v2/bav:BSG-OBJ-0000000000000117/info.json`

That endpoint returns IIIF metadata, and the derived full-image request returns a JPEG. So this is not an unsupported protocol; it is a question about the correct current IIIF URL for the object.

I labeled this `question` and left it open because automatic discovery from the Bavarikon object page may still require site-specific extraction: the page exposes the IIIF base/API data rather than the exact `/info.json` URL in the simple form Dezoomify currently detects.
